### DISCUSSION TITLE
Maquina desconhecida no meu painel de gerenciamento

### DISCUSSION THREAD

**Discussion by Givannixs** [amidaware/tacticalrmm#1866] [Q&A] [answered]
Acredito ter identificado uma falha de segurança crítica no Tactical RMM. Ao instalar o agente RMM em uma das máquinas do meu parque, ele também instala uma máquina aleatória com o nome DESKTOP-9981ETL e o IP 35.223.75.0. Desconheço essa máquina. Ao acessá-la para verificar do que se trata, parece ser uma VM do Google, cujo proprietário desconheço. Quando a pessoa percebeu que fiz acesso, desligou a máquina com receio de ser alvo de um ataque, pelo que suspeito. Minha preocupação é: se consegui acessar a máquina de outra pessoa, quem garante que eles não podem acessar as máquinas presentes no meu painel também?

Entrei em contato com vocês em busca de ajuda para entender o que está acontecendo.

![Screenshot_2](https://github.com/amidaware/tacticalrmm/assets/128402754/594d539b-a5c8-41a9-97ea-bd53132e797c)

**Comment by dinger1986** [comment, marked as answer]
https://docs.tacticalrmm.com/faq/#help-ive-been-hacked-and-there-are-weird-agents-appearing-in-my-tactical-rmm
